test/api_integration/apis/index_patterns/runtime_fields_crud/get_runtime_field/errors.ts

```diff
@@ -67,15 +67,5 @@ export default function ({ getService }: FtrProviderContext) {
         '[request params.id]: value has length [1759] but it must have a maximum length of [1000].'
       );
     });
-
-    it('returns error when attempting to fetch a field which is not a runtime field', async () => {
```

Comment:
No need of this test as the lookup is in the runtimeFieldMap and not the field list, so there is no way to get a non-runtime field through API.